source/main/environment.js

```diff
@@ -2,7 +2,7 @@
 import os from 'os';
 import { uniq, upperFirst, get, includes } from 'lodash';
 import { version } from '../../package.json';
-import { stateDirectoryPath } from './config';
+// import { stateDirectoryPath } from './config';
```

Comment:
@DeeJayElly we can not use anything from `./config` as this breaks the storybook.
The problem is that within `./config` we use `fs` to load the `LauncherConfig` which contains all the paths we need (incl. the state path).

Reply:
This import should be removed and we should send the state path to the renderer process through the IPC channel. Similar to how we do for the system locale: https://github.com/input-output-hk/daedalus/blob/develop/source/main/index.js#L142
Network status store should receive this value on app start time and then propagate it to the Network status container and component.